packages/ui/v2/core/Badge.tsx

```diff
@@ -24,17 +24,19 @@ export type BadgeProps = {
   size?: keyof typeof classNameBySize;
   StartIcon?: Icon;
   bold?: boolean;
+  rounded?: boolean;
```

Comment:
Is this new property to have rounded badges needs to be added somewhere else expecting rounded badges now that it doesn't come by default?

Reply:
its fine, rounded is only for small badges such as numbered badges